securedrop/journalist.py

```diff
@@ -286,7 +288,7 @@ def commit_account_changes(user):
                                                                           e))
             db_session.rollback()
         else:
-            flash("Account successfully updated!", "success")
+            flash("Account updated.", "success")
```

Comment:
I think "success" is redundant here.

Reply:
success is outside of the string displayed to the user, it is the last argument of the flash function and determines how to style the message.